src/Sylius/Bundle/CoreBundle/Controller/UserController.php

```diff
@@ -21,8 +25,150 @@ class UserController extends ResourceController
      */
     public function filterFormAction(Request $request)
     {
-        return $this->render('SyliusWebBundle:Backend/User:filterForm.html.twig', array(
-            'form' => $this->get('form.factory')->createNamed('criteria', 'sylius_user_filter', $request->query->get('criteria'))->createView()
-        ));
```

Comment:
Does it make more readable? Not really. :) I'd extract it to separate method if needs change at all.